test/common/matchers.dart

```diff
@@ -5,6 +5,7 @@ import 'package:test/test.dart';
 
 import 'package:build/build.dart';
 
+final argumentError = new isInstanceOf<ArgumentError>();
```

Comment:
Exists in `package:matcher`

https://github.com/dart-lang/matcher/blob/master/lib/src/error_matchers.dart#L9